### ISSUE TITLE
Feature: Decode Text to Binary

### ISSUE THREAD

**Issue by CodeSpoof** [wierkstudio/ciphereditor#34]
### Describe your feature or enhancement suggestion
Currently it's only possible to encode Binary to Text not the other way.

### Describe a concrete use case
Analyze .onion addresses, since they are base32 encoded and contain public key and checksum.

**Comment by ffraenz** [comment]
Hi @CodeSpoof! In most operations available on ciphereditor you can feed in values the other way round for decoding or decryption. In the case of the "Binary to text encoding" operation, change the value of the "Encoded Data" control and you will see the result appear in the "Data" Control. Does that work for you?

**Comment by jphastings** [comment]
I had the same confusion when I first used your tool — there's something about the UI that makes it hard to recognise that you can run each block either way. Perhaps even a _purely cosmetic_ "flip" button for blocks that swaps the top and bottom fields would give new users enough confidence that blocks _don't_ run "top to bottom" (which was my assumption).

**Comment by ffraenz** [comment]
Yes, I thought of that as well, this is how it is currently being done on cryptii, having the encode/decode or encrypt/decrypt toggle button.
